frontend/reactnative/App.js

```diff
@@ -2,7 +2,7 @@ import React, {Component} from 'react';
 import {Text, View, StyleSheet} from 'react-native';
 import {Location, Permissions, MapView} from 'expo';
 
-import * as Config from './lib/Config.js';
```

Comment:
Do we want to be removing this line?
I know its included in Auth.js but say it's removed from Auth.js it can break the app without much warning

Reply:
Config is not used in the class, so it is not neccicary to import it.